franklin/github/api.py

```diff
@@ -106,23 +104,4 @@ def get_access_token(request):
         "redirect_uri": request.data.get('redirectUri'),
         "client_secret": os.environ['SOCIAL_AUTH_GITHUB_SECRET']
     }
-
-    # Exchange authorization code for access token.
-    r = make_rest_post_call(url, headers, params)
-    if status.is_success(r.status_code):
-        try:
-            access_token = r.json().get('access_token', None)
-            user = do_auth(access_token)
-            serializer = UserSerializer(user)
-            response_data = Response({
-                'token': access_token,
-                'user': serializer.data
-            }, status=status.HTTP_200_OK)
-        except KeyError:
-            response_data = Response({'status': 'Bad request',
-                                      'message': 'Authentication could not be\
-                                              performed with received data.'},
-                                     status=status.HTTP_400_BAD_REQUEST)
-        return response_data
-    else:
-        return Response(status=r.status_code)
+    return make_rest_post_call(url, headers, params)
```

Comment:
Since this file is now a Github SDK, I've moved all app specific logic up into `view.py`.